homeassistant/components/starline/sensor.py

```diff
@@ -30,47 +31,56 @@
         translation_key="battery",
         device_class=SensorDeviceClass.VOLTAGE,
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
         key="balance",
         translation_key="balance",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
```

Comment:
What is the state of this entity?

Reply:
It's SIM card balance

Reply:
But there is no unit of measurement attached to this

Reply:
It's dynamic, AFAICS: https://github.com/home-assistant/core/blob/c30a0ba34d69c316025f5bb676d65e461b21282f/homeassistant/components/starline/sensor.py#L152-L155